src/Core/Install.php

```diff
@@ -444,24 +443,23 @@ public function checkSmarty3()
 	 *
 	 * Checks, if "url_rewrite" is enabled in the ".htaccess" file
 	 *
-	 * @param string $basepath   The basepath of the app
 	 * @param string $baseurl    The baseurl of the app
 	 * @return bool false if something required failed
 	 */
-	public function checkHtAccess($basepath, $baseurl)
+	public function checkHtAccess($baseurl)
 	{
 		$status = true;
 		$help = "";
 		$error_msg = "";
 		if (function_exists('curl_init')) {
-			$fetchResult = Network::fetchUrlFull($basepath . "/install/testrewrite");
+			$fetchResult = Network::fetchUrlFull($baseurl . "/install/testrewrite");
 
 			$url = normalise_link($baseurl . "/install/testrewrite");
-			if ($fetchResult->getBody() != "ok") {
+			if ($fetchResult->getReturnCode() != 204) {
 				$fetchResult = Network::fetchUrlFull($url);
 			}
 
-			if ($fetchResult->getBody() != "ok") {
+			if ($fetchResult->getReturnCode() != 204) {
 				$status = false;
 				$help = L10n::t('Url rewrite in .htaccess is not working. Check your server configuration.');
```

Comment:
Shouldn't this be `Make sure you copied .htaccess-dist to .htaccess`?

Reply:
We can add it too :-)